bigbone/src/main/kotlin/social/bigbone/api/entity/Report.kt

```diff
@@ -73,9 +73,15 @@ data class Report(
     /**
      * Specify the typology of category among spam, violation or other.
      */
-    enum class ReportType(val type: String) {
-        SPAM("spam"),
-        VIOLATION("violation"),
-        OTHER("other")
+    @Serializable
+    enum class ReportType {
```

Comment:
Maybe this should be renamed `ReportCategory` to match the [documentation](https://docs.joinmastodon.org/entities/Report/#category) where it’s a `category` parameter?